File: drink/src/main/java/cafeteria/Ordered.java
package cafeteria;

public class Ordered extends AbstractEvent {

    private Long id;
    private String productName;
    private String phoneNumber;
    private Integer qty;
    private Integer amt;
    private String status;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
    public String getProductName() {
        return productName;
    }

    public void setProductName(String productName) {
        this.productName = productName;
    }
    public String getPhoneNumber() {
    	return phoneNumber;
    }
    public void setPhoneNumber(String phoneNumber) {
    	this.phoneNumber = phoneNumber;
    }
    public Integer getQty() {
        return qty;
    }

    public void setQty(Integer qty) {
        this.qty = qty;
    }
    public Integer getAmt() {
        return amt;
    }

    public void setAmt(Integer amt) {
        this.amt = amt;
    }
    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: customercenter/src/main/scala/cafeteria/Ordered.scala
package cafeteria

import scala.beans.BeanProperty

class Ordered extends AbstractEvent {
  @BeanProperty
  var id :Long = 0L
  @BeanProperty
  var phoneNumber :String = null
  @BeanProperty
  var productName :String = null
  @BeanProperty
  var qty :Integer = 0
  @BeanProperty
  var amt :Integer = 0
  @BeanProperty
  var status :String = null
}